### DISCUSSION TITLE
CaptchaChallenge. How use proxy ?)

### DISCUSSION THREAD

**Discussion by blackkvadrat** [xtekky/gpt4free#2090] [Q&A]
I don't understand why I get the error CaptchaChallenge: Use other cookies or/and ip address. To run gf4, I use my own proxy, which I placed on a rented VPS. I made my proxy using tinyproxy and for the test I did not use a login and password to log in and also allowed access from any IP. I don't understand what i m doing wrong ) 
![screenshot](https://github.com/xtekky/gpt4free/assets/36846738/6614b549-37db-4034-9744-e9a3e86307d6)

**Comment by W1Real** [comment]
I'm also having issue, but for the UI executable on windows. It seems like it's not going through my proxy, even though I already set the environment variable. `G4F_PROXY=http://host:port`

Btw, the best free proxy is opera-proxy: https://github.com/Snawoot/opera-proxy
